uber/tests/models/test_badge_funcs.py

```diff
@@ -446,20 +453,3 @@ def test_out_of_badge_type(self, session, monkeypatch, before_printed_badge_dead
         session.regular_attendee.badge_num = None
         assert 'There are no more badges available for that type' == check(session.regular_attendee)
 
-
-class TestDupeFixes:
```

Comment:
We're removing support for 'fixing' duplicate numbers because, thanks to our unique constraint, it's now actually impossible to have them.